openapi.yaml

```diff
@@ -562,7 +562,7 @@ components:
           
   schemas: {}
 servers:
-  - url: 'https://sandbox.bluebutton.cms.gov/'
+  - url: 'http://localhost:8000/'
```

Comment:
I don't think this should be changed to localhost

Reply:
provided multiple urls for SBX, TEST, and Local since we need to test it out on local dev and LLE when changes needed and need to test it out on local and LLE before deploy....